components/Typography/src/MDCTypography.m

```diff
@@ -177,4 +178,25 @@ - (UIFont *)italicFontOfSize:(CGFloat)fontSize {
   return [UIFont italicSystemFontOfSize:fontSize];
 }
 
+- (BOOL)isLargeForContrastRatios:(UIFont *)font {
+  if (font.pointSize >= 18) {
+    return YES;
+  }
+  if (font.pointSize < 14) {
+    return NO;
+  }
+
+  UIFontDescriptor *fontDescriptor = font.fontDescriptor;
+  if ((fontDescriptor.symbolicTraits & UIFontDescriptorTraitBold) == UIFontDescriptorTraitBold) {
+    return YES;
+  }
+
+  // We treat medium as large for MDC accesibility when larger than 14
+  if ([font.fontName rangeOfString:@"medium" options:NSCaseInsensitiveSearch].location != NSNotFound) {
```

Comment:
My bad—I missed that this was in MDCSystemFontLoader. Consider splitting it out.